frontend/src/store/playbackManager.ts

```diff
@@ -101,88 +111,89 @@ const defaultState: PlaybackManagerState = {
   playbackInitMode: InitMode.Unknown
 };
 
-const state = reactive<PlaybackManagerState>(defaultState);
+export const mediaElementRef = ref<HTMLMediaElement>();
+const state = reactive<PlaybackManagerState>(cloneDeep(defaultState));
+const mediaControls = useMediaControls(mediaElementRef);
+const reactiveDate = useNow();
+/**
+ * Previously, we created a new MediaMetadata every time the item changed. However,
+ * that made the MediaSession controls disappear for a second. Keeping the metadata
+ * as a global variable and updating it solves this problem.
+ */
+const mediaMetadata = new MediaMetadata();
 
 class PlaybackManagerStore {
   /**
    * == GETTERS ==
    */
-  public get status(): typeof state.status {
+  public get status(): PlaybackStatus {
     return state.status;
   }
```

Comment:
I don't remember, can getters be arrows functions?

Reply:
It can't, at leat it gave me syntax errors.

Reply:
Np :)